src/components/pagination/index.md.njk

```diff
@@ -9,81 +9,61 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-Help the user navigate a series of pages like search results or guides.
+Help users navigate forwards and backwards through a series of pages. For example, search results or guidance that's divided into multiple website pages - like [the GOV.UK mainstream guide format](https://govuk-prototype-kit.herokuapp.com/docs/templates/mainstream-guide).
```

Comment:
Add em-dash (—).
Also, the link is coming up broken for me?

```suggestion
Help users navigate forwards and backwards through a series of pages. For example, search results or guidance that's divided into multiple website pages — like [the GOV.UK mainstream guide format](https://govuk-prototype-kit.herokuapp.com/docs/templates/mainstream-guide).
```

Reply:
Ahh I can see what you're trying to link to:
https://github.com/alphagov/govuk-prototype-kit/blob/main/docs/views/templates/mainstream-guide.html

I'll ask the Prototype team for help.

Reply:
Link is working now